src/web/crate_details.rs

```diff
@@ -76,11 +77,11 @@ pub struct Release {
 
 impl CrateDetails {
     pub fn new(
-        conn: &mut Client,
+        conn: &mut impl GenericClient,
```

Comment:
Was gonna say this should probably be `conn: impl GenericClient`, but `postgres` is missing the `impl GenericClient for &mut impl GenericClient` forwarding impl 😭